nixos/doc/manual/release-notes/rl-1703.xml

```diff
@@ -217,6 +217,18 @@ following incompatible changes:</para>
     </para>
   </listitem>
 
+  <listitem>
+    <para>
+    Iputils are no longer provide ping6 and traceroute6. The functionality of
+    these tools were integrated respectively into ping and traceroute. To
```

Comment:
were -> have been, respectively into ping and traceroute -> into ping and traceroute respectively